### PR TITLE
Add CPU % usage to Info Panel

### PR DESCRIPTION
**Pull request by adrianbj**

### CODE REVIEW

**Comment by adrianbj** [comment]
screen shot 2016-02-24 at 10 09 27 am

**Comment by dg** [comment]
What about to add both information, user usage and system usage?

File: src/Tracy/Debugger.php [hunk 1]
```diff
@@ -135,6 +141,13 @@ public static function enable($mode = NULL, $logDirectory = NULL, $email = NULL)
 		self::$obLevel = ob_get_level();
 		error_reporting(E_ALL);
 
+		// get initial rusage for CPU output in info panel
+		self::$getrusage = function_exists('getrusage');
+		if(self::$getrusage) {
+			$dat = getrusage();
+			self::$rusage = $dat["ru_utime.tv_sec"]*1e6+$dat["ru_utime.tv_usec"];
+		}
```
Comment (enumag) [inline src/Tracy/Debugger.php]: Shouldn't this be skipped in production mode? Also your coding style is not correct.

**Comment by adrianbj** [comment]
Sorry for the incorrect coding style - I assume you meant the location of curly braces? I'll be sure to pay more attention to this in any future PR's.

Thanks for implementing the system usage as well.

**Pull request closed without merging**